src/dlgConnectionProfiles.cpp

```diff
@@ -36,6 +36,7 @@
 dlgConnectionProfiles::dlgConnectionProfiles(QWidget * parent)
 : QDialog( parent )
 , mProfileList( QStringList() )
+, offline_button( Q_NULLPTR )
```

Comment:
Actually,  we have been phasing out the use of the Qt specific macro `Q_NULLPTR` to go to the more generic (in later C++ standards) `nullptr`.

:bulb: Now I have made that into a *do-able* issue: #2594.